### PR TITLE
Updating README.md

### PR DESCRIPTION
**Pull request by chipoglesby**
Hi @chooliu, here are some updates to your amazing work so far! Thank you for doing this.

I've added the following changes:

1. Fixing broken links
2. Adding a table with columns: 
  a. Site
  b. Owner
  c. Description
  d. Data Type
3. Adding the following headers for sources:
  a. State Organizations
  b. County Organizations
  c. City Organizations
  d. Citizen Wrangled

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,36 +2,50 @@
...
+| [hub.arcgis.com/datasets?q=denver](http://hub.arcgis.com/datasets?q=denver) | ArcGIS Hub | | |
+| [lexisnexis.com/hottopics/colorado](http://lexisnexis.com/hottopics/colorado) | Colorado Constitution, Revised Statues, Court Rules | | |
+| [geodata.co.gov](https://geodata.co.gov/) | Colorado GeoData Cache | | |
+| [rtd-denver.com/Developer.shtml](http://rtd-denver.com/Developer.shtml) | Regional Transportation District | | |
+| [sos.state.co.us](http://sos.state.co.us) | Secretary of State | | |
+| [water.state.co.us/DataMaps/DataSearch/Pages/DataSearch.aspx](http://water.state.co.us/DataMaps/DataSearch/Pages/DataSearch.aspx) | Department of Natural Resources | | |
+
+## County Organizations With Open Data
+| Site | Owner/County | Description | Data Type |
+|:----|:---|:---|:----|
```
Comment (andypicke) [inline README.md]: Add City of Boulder open data: https://bouldercolorado.gov/open-data ?

File: README.md [hunk 2]
```diff
@@ -2,36 +2,50 @@
...
+| | | | |
+
+## City Organizations With Open Data
+| Site | Owner/City | Description | Data Type |
+|:----|:---|:---|:----|
+| | | | |
```
Comment (andypicke) [inline README.md:43]: Add city of Boulder open data site: https://bouldercolorado.gov/open-data ?

**Comment by andypicke** [comment]
@chipoglesby This looks great, thanks for organizing. I don't have write access, so someone else will have to approve though.

**Comment by chooliu** [comment]
merging #2 , which addresses part of issue #1 !

**Comment by chooliu** [comment]
Now to add some stuff like City of Boulder's data portal, etc to the awesome table framework @chipoglesby made!

@andypicke feel free to add Boulder in a pull request if you feel like practicing Git, otherwise I'll add it later 👍

File: README.md (hunk 2 above)
Comment (chipoglesby) [reply README.md:43]: @andypicke I can add you as a contributor to my repo if you'd like. @chooliu how do you handle those types of requests?

**Comment by chipoglesby** [comment]
Thank you @andypicke and @chooliu for the feedback!

File: README.md (hunk 2 above)
Comment (andypicke) [reply README.md:43]: I just made my own fork after @chooliu merged yours, and added my lines. I think that's the proper way to do it (relatively new to collaborating on Github) ?

**Pull request merged**